tests/unit/models/test_utils.py

```diff
@@ -11,3 +32,24 @@ class TestUtils(TestCase):
     def test_kwargs_req(self):
         with self.assertRaises(XRPLModelException):
             IssuedCurrency(currency, issuer)
+
+    def test_throws_if_positional_args_mixed_with_non_positional_args(self):
+        with self.assertRaises(XRPLModelException):
+            ChannelAuthorize(
+                20,
+                True,
+                channel_id=_CHANNEL_ID,
+                amount=_AMOUNT,
+                passphrase=_DUMMY_STRING,
+                seed_hex=_DUMMY_STRING,
+            )
+
+    def test_positional_args_in_ledger_objects_constructor_throws(self):
```

Comment:
I'm confused by this test name - `Sign` is not a ledger object, it's an RPC method.